README.md

```diff
@@ -121,6 +122,33 @@ options:
     timezone:
     --------
     Available when you set type to date. You can configure datetime's timezone with this property.
+undefinedable:
+--------------
+Make property undefinedable. This property makes you able to find undefined properties.
+Example:
+
+```php
+namespace Metglobal\Compass\Request;
+
+use Metglobal\DTOBundle\Annotation\Parameter;
+use Metglobal\DTOBundle\Request;
+use Metglobal\DTOBundle\Undefined;
+
+class DummyRequest implements Request
+{
+    /**
+    * @Parameter(scope="query", undefinedable=true)
+    *
+    * @var string|Undefined
+    */
+    public $xyzProperty;
+}
+```
+Results would be:
+
+| ?xyzProperty= | ?xyzProperty=dto | ?otherProperty=5 |
```

Comment:
şu kısım daha açıklayıcı olabilir. örn: 
`?otherProperty=5` ise Undefined()
`?otherProperty=5&xyzProperty=` null gibi.

Reply:
Aynı şey yazıyor esasen. Query string olduğu da aşkar, bu konuda katılmıyorum sana

Reply:
esasen sondaki otherProperty örneğinde param. olmadığını göstermek amacıyla düşünmüştüm, birinde belirttik birinde belirtmedik gibisinden.

Reply:
En sondakinde xyzProperty göndermediğimiz için ? bırakacaktık. Bu daha anlaşılmaz oluyor diye farklı birşey yazdım aslında, extra olsun diye değil